packages/ui-toggle-details/src/ToggleDetails/index.tsx

```diff
@@ -102,15 +99,13 @@ class ToggleDetails extends Component<ToggleDetailsProps> {
     )
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'toggleProps' implicitly has an 'any' ty... Remove this comment to see the full error message
-  renderToggle(toggleProps, expanded) {
+  renderToggle(toggleProps: any, expanded: boolean) {
```

Comment:
Since this is coming from the render method of Expandable, I suggest to export the `GetToggleProps` type from there and use here. I would rename it to `ExpandableToggleProps` too.